src/sentry/migrations/0203_semver_backfill.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 1.11.29 on 2021-05-18 01:44
+from django.db import connection, migrations
+from psycopg2.extras import execute_values
+from sentry_relay.exceptions import RelayError
+from sentry_relay.processing import parse_release
```

Comment:
This seems like a long term risk, as this import could move.

Reply:
It's not ideal, but I'd prefer to use the parser to produce this data. The code is written in rust, so it's not easy to just copy it over.